### PR TITLE
Remove scrollbar for non-official rooms

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by ghost** [comment]
Why are you getting rid of slowchat?

**Comment by DeathlyPlays** [comment]
That was accidental tbh

**Comment by DeathlyPlays** [comment]
Dont you just love coding on mobile

**Comment by DeathlyPlays** [comment]
Done

**Comment by Lord-Haji** [comment]
Ohh God Someone save this kid @DeathlyPlays . No Such CSS Class called ``infobox-unlimited`` exists the client. Use ``infobox``

**Comment by DeathlyPlays** [comment]
It works @Lord-Haji 

_HoeenCoder edit: fix accidental user tag_

**Comment by ghost** [comment]
@DeathlyPlays you failed to @Lord-Haji properly

File: rooms.js [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ class ChatRoom extends Room {
 	}
 	getIntroMessage(user) {
 		let message = '';
-		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + (!this.isOfficial ? ' class="infobox-limited"' : '') + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
+		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + (!this.isOfficial ? ' class="infobox-unlimited"' : '') + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
```
Comment (HoeenCoder) [inline rooms.js]: Just remove the offical check, not add a non-existant CSS class.

**Review by HoeenCoder — COMMENTED** [review]
Pending policy approval.

File: rooms.js (hunk 1 above)
Comment (DeathlyPlays) [reply rooms.js]: ok

File: rooms.js [hunk 2]
```diff
@@ -1350,7 +1350,7 @@ class ChatRoom extends Room {
 	}
 	getIntroMessage(user) {
 		let message = '';
-		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + (!this.isOfficial ? ' class="infobox-limited"' : '') + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
+		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
```
Comment (HoeenCoder) [inline rooms.js:1353]: `<div' + '>'` -> `<div>`

**Comment by Mystifi** [comment]
I think this could be something purchased from the shop. I definitely don't want all non-official rooms to have it, but I'd be fine if they purchased it or something. Thoughts, Hoeen?

**Comment by HoeenCoder** [comment]
Thats a pretty good comprimise actually, since im not too keen on this being default.

**Comment by ghost** [comment]
hmm, how can you code it if only lets just say,,,, only 2 rooms buy and use it? becuase in that code it applies to all rooms

**Comment by Lord-Haji** [comment]
@Stellation nothing is impossible

@HoeenCoder  I Could Write the Code

P.S - The Private Shop will need its addition 📦

**Comment by HoeenCoder** [comment]
Its not hard to add it to the shop, and when its bought (and when its set for the selected room), add a tag to the chatroom data which will cause the intro to not be limited. Meanwhile this code would need an edit to check said tag to see if the room is either official, or if someone bought this for the room. If either of those is correct, dont limit, otherwise use the normal limit.

**Comment by Lord-Haji** [comment]
@Mystifi @HoeenCoder @Stellation @DeathlyPlays 
**Refer to #193 for further changes**

**Comment by DeathlyPlays** [comment]
e.e

**Pull request closed without merging**